src/Windowing/Silk.NET.GLFW/Silk.NET.GLFW.csproj

```diff
@@ -8,11 +8,16 @@
     
     <ItemGroup>
       <PackageReference Include="AdvancedDLSupport" Version="3.0.0" />
-      <PackageReference Include="opentoolkit.redist.glfw" Version="3.3.0-pre20190424184726" />
       <PackageReference Include="Ultz.Dispatcher" Version="1.1.0" />
     </ItemGroup>
     
     <ItemGroup>
       <ProjectReference Include="..\..\Silk.NET.Core\Silk.NET.Core.csproj" />
     </ItemGroup>
+
+    <ItemGroup>
+        <Content Include="..\..\..\build\native\glfw\*">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </Content>
+    </ItemGroup>
```

Comment:
Add the following lines to the ItemGroup in this csproj:
```xml
<None Include="Silk.NET.GLFW.targets" PackagePath="build/" />
```

And then make a file called Silk.NET.GLFW.targets (in the main project dir) with the following contents:
```xml
<Project>
  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
    <None Include="$(MSBuildThisFileDirectory)\..\runtimes\win-x64\native\glfw3-x64.dll">
      <Link>glfw3-x64.dll</Link>
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      <Visible>false</Visible>
    </None>
    <None Include="$(MSBuildThisFileDirectory)\..\runtimes\win-x86\native\glfw3-x86.dll">
      <Link>glfw3-x86.dll</Link>
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      <Visible>false</Visible>
    </None>
  </ItemGroup>
</Project>
```